src/components/searchbutton/SearchButton.scss

```diff
@@ -2,12 +2,12 @@
   position: relative;
   height: 120px;
   width: 110px;
-  border-radius: 7px;
+  border-radius: 5px;
   outline: 0;
   border: none;
   cursor: pointer;
   margin-top: 10px;
-  transition: 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
+  //transition: 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
```

Comment:
I'd actually keep this uncommented. It makes the background color change smoother. Also the reason the background color doesn't slide up is cuz the naming got changed (iconContainer --> icon) and iconContainer wrappers were deleted since they weren't needed anymore. But I kinda like the simplicity of no slide up, so we can just keep it as it

Reply:
but if you want the slide up then I can help getting it back. If no slide, then remove all the &:before classes below. I think they were only there for the animation